src/composables/notifications.ts

```diff
@@ -0,0 +1,227 @@
+import {
+  computed, onBeforeUnmount, onMounted, ref,
+} from '@vue/composition-api';
+import {
+  IAccount, IChainNames,
+  INetwork,
+  INotification,
+  INotificationSetting,
+  NotificationStatus,
+} from '../types';
+import { useGetter, useState } from './vuex';
+import {
+  NOTIFICATION_STATUS_CREATED,
+  NOTIFICATION_STATUS_READ,
+  NOTIFICATION_TYPE_WALLET,
+  executeAndSetInterval,
+  watchUntilTruthy, fetchJson, postJson, NOTIFICATION_ENTITY_TYPE_TIP, AGGREGATOR_URL,
+} from '../popup/utils';
+
+const fetchedNotificationsLimit = 20;
+
+const mountedComponentsCounter = ref<number>(0);
+const notificationsSuperhero = ref<INotification[]>([]);
+const notificationsWallet = ref<INotification[]>([]);
+let pollingInterval: NodeJS.Timer | null = null;
+
+export function useNotifications(preventMountedIncrement = false) {
```

Comment:
I think this composable's settings can grow over time so I suggest to use the common pattern of providing the settings as an object. You can see this here: src/composables/tokensList.ts

Also I suggest to inverse the logic of this setting. For example:
```
requirePolling: false // Default value
```
So if a component needs the actual notifications this setting should be set to `true`.
What do you think?